### DISCUSSION TITLE
MS%26E%40Stanford/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#124] [Announcements]
# MS%26E%40Stanford/

https://csmsapp.org/MS%26E%40Stanford/

**Comment by JamesSand** [comment]
求一些 data points 或者申请实时信息 qvq

**Reply by HUANGLIZI** [reply]
你可以参考datapoints列表

**Comment by KingArthur0205** [comment]
很多录取人都是工作几年，变成senior engineer/创业等管理经验后去申请。new grad慎重申请
